Eatery/Controllers/Eateries/Campus/ScrollableViewController.swift

```diff
@@ -0,0 +1,374 @@
+//
+//  ScrollableViewController.swift
+//  Eatery
+//
+//  Created by Sergio Diaz on 12/9/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ScrollableViewController: UIViewController {
+
+    private var eatery: CampusEatery!
+    private var viewControllers: [CampusEateryExpandedMenuViewController]!
+    private var allItems: [ExpandedMenu.Item]!
+
+    private var originalViews: [UIView]!
+    private var tabBar: TabBar!
+    private var tabBarStack: UIStackView!
+    private var menuItemsStack: UIStackView!
+    private var menuItemsHighToLowStack: UIStackView!
+    private var menuItemsLowToHighStack: UIStackView!
+
+    private var menuItemsView: UIView!
+    private var filterButton: ExtendedFilterButton!
+    private var filterLabel: UILabel!
+    private var headerContainer: UIView!
+    private var separatorView: SeparatorView!
+
+    let tabBarPadding: CGFloat = 16
+    var scrollView: UIScrollView?
+    var scrollOffset: CGFloat = 0
+    var fillerSpaceHeight: CGFloat = 0
```

Comment:
Is it possible to make these private?

Reply:
To avoid messiness, I didn't want to place the scrollView or the scrollOffset in the constructor.  However, ScrollableViewController still on the scrollOffset and scrollView of its parent view controllers, so within `CampusEateriesViewController`, I actually access the variables publicly and set them in there.  That's why I didn't make those private.  TabBarPadding and filterSpaceHeight can be made private though

Reply:
Would you rather me create a setter method in ScrollableViewController that would set the scrollView and scrollOffset and allow them to 1. not be in the constructor to avoid messiness and 2. be private variables?

Or would you prefer some other method of going about this?

Reply:
If there's a reason to let them be `internal` then that's good enough for me